README.md

```diff
@@ -91,6 +91,8 @@ Then, set the `RISCV` env variable as:
 export RISCV=/home/$USER/tools/riscv
 ```
 
+Optionally you can also compile with clang/LLVM instead of gcc. For that you must install the clang compiler into the same `RISCV` path. The binaries of gcc and clang do not collide so you can have both residing in the same `RISCV` directory. For this you can set the `-DCMAKE_INSTALL_PREFIX` cmake variable to `$RISCV` when building LLVM.
```

Comment:
This will depend on the llvm build that each person needs, I don't believe there is a generic way as for gcc. In my case I used a modified version of LLVM (https://github.com/artecs-group/llvm-xposit) but the install outputs should be the same for other builds.

Reply:
Ok, I can add a generic way of installing LLVM with clang for RISC-V targeting the $RISCV for instalation